Localization/sv-SE/together-we-are-powerful.po

```diff
@@ -0,0 +1,191 @@
+msgid ""
+msgstr ""
+"FAH-Translate-Schema-Version: 1.0\n"
+"URL: https://foldingathome.org/together-we-are-powerful/\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Source-Language: en-US\n"
+"Source-Version: 1.2\n"
+"Source-Last-Modified: 20200904\n"
```

Comment:
This doesn't match with the file on en-US version. 
Where it says:
"Source-Version: 1.1\n"
"Source-Last-Modified: 20200906\n"